logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/AvsClient.kt

```diff
@@ -0,0 +1,25 @@
+package com.wire.kalium.logic.data.call
+
+data class AvsClient(
+    val userId: String,
+    val clientId: String
+)
+
+data class AvsClientList(
+    val clients: List<AvsClient>
+) {
+    fun toJsonString(): String {
+        val start = "{"
+        var startClients = "clients:["
+
+        clients.forEachIndexed { index, avsClient ->
+            startClients += if (index > 0) "," else ""
+            startClients += "{userid: ${avsClient.userId},clientid: ${avsClient.clientId}}"
```

Comment:
Also missing quotes here. I suggest writing a Unit test for this.

Reply:
Yup yup, changed it on your function suggestion!